README.md

```diff
@@ -161,6 +161,8 @@ or add the following to your `cypress.json`
 - `customSnapshotsDir` : Path to the directory that snapshot images will be written to, defaults to `<rootDir>/cypress/snapshots`.
 - `customDiffDir`: Path to the directory that diff images will be written to, defaults to a sibling `__diff_output__` directory alongside each snapshot.
 
+- `retryCounter` : Retry up to n times before raise a failer and the execution gonna wait for 100 millisecond betwen each iteration.
```

Comment:
slightly better grammar:
`Retry up to n times before raise a failure. The process will wait for 100 millisecond between each iteration.`

and, if 100 becomes a configurable default instead of a constant, insert "by default" before final period

Reply:
done